dev-tools/openapi-manager/src/check.rs

```diff
@@ -0,0 +1,185 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use std::{io::Write, process::ExitCode};
```

Comment:
A lot of this code is quite generic, and I think can live separately. What do you think about moving this out to the dropshot repo or its own repo?

Reply:
In a meeting yesterday we decided to wait and see -- leave it in omicron for now but be open to the possibility of putting this code in a central location later.